Added more outputs

keb721 opened the pull request:
Allowed code to output positions as inputs for GROMACS
and LAMMPS (with and without M-site) in addition to input
for DL_POLY.

Updated README.md to reflect new command line option.

Commit:
Added more outputs

Allowed code to output positions as inputs for GROMACS
and LAMMPS (with and without M-site) in addition to input
for DL_POLY.

Updated README.md to reflect new command line option.

dquigley-warwick commented:
I'm being uber-picky here, but...

`if (foundcount==N*third) exit`

is a bit risky. The parameter `third` is a double precision real and so `N*third` will be double precision. You're then doing an exact comparison of an integer to a floating point number, which I think involves promoting the integer to the nearest double precision real number. 

It'll work in this case (trivial algebra), but exact comparison of two floating point numbers should generally be avoided as they won't generally agree if computed via different sequences of operation due to round-off.

The alternative

` if (foundcount==N/3) exit`

is an exact comparison between two integers and will be satisfied the first time we've found all the oxygen atoms.

Commit:
Fixed incorrect promotion to float

Reverted if statement to exact comparison between two
integers, instead of potentially spurious comparison between
two floats.

keb721 commented:
Thanks for that - I hadn't considered the possible issue!
